test/config.spec.coffee

```diff
@@ -60,8 +60,6 @@ describe 'AWS.Config', ->
       expect(configure(maxRetries: 2).maxRetries).to.equal(2)
 
   describe 'retryDelayOptions', ->
-    it 'defaults to "base: 100"', ->
```

Comment:
There is a separate test that implicitly tests this, but happy to add an explicit test.